Libraries/Opc.Ua.Client/TraceableSession.cs

```diff
@@ -390,11 +390,11 @@ public ReferenceDescription FindDataDescription(NodeId encodingId)
         }
 
         /// <inheritdoc/>
-        public Task<DataDictionary> FindDataDictionary(NodeId descriptionId)
+        public async Task<DataDictionary> FindDataDictionary(NodeId descriptionId)
```

Comment:
Actually, this change is not necessary. The caller of this method needs to await it.

Reply:
Ah, I retract my comment. When using "using", then this is a must.